[2022-11-27T16:01:43.445+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dag_youtube_api.getDataToLocal manual__2022-11-27T16:01:41.037302+00:00 [queued]>
[2022-11-27T16:01:43.458+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dag_youtube_api.getDataToLocal manual__2022-11-27T16:01:41.037302+00:00 [queued]>
[2022-11-27T16:01:43.459+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-27T16:01:43.459+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 2
[2022-11-27T16:01:43.459+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-27T16:01:43.479+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): getDataToLocal> on 2022-11-27 16:01:41.037302+00:00
[2022-11-27T16:01:43.498+0000] {standard_task_runner.py:55} INFO - Started process 8469 to run task
[2022-11-27T16:01:43.510+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dag_youtube_api', 'getDataToLocal', 'manual__2022-11-27T16:01:41.037302+00:00', '--job-id', '363', '--raw', '--subdir', 'DAGS_FOLDER/dag_youtube_api.py', '--cfg-path', '/tmp/tmprp_ixxuo']
[2022-11-27T16:01:43.518+0000] {standard_task_runner.py:83} INFO - Job 363: Subtask getDataToLocal
[2022-11-27T16:01:43.652+0000] {task_command.py:376} INFO - Running <TaskInstance: dag_youtube_api.getDataToLocal manual__2022-11-27T16:01:41.037302+00:00 [running]> on host 1977dd9b5152
[2022-11-27T16:01:43.797+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dag_youtube_api
AIRFLOW_CTX_TASK_ID=getDataToLocal
AIRFLOW_CTX_EXECUTION_DATE=2022-11-27T16:01:41.037302+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-11-27T16:01:41.037302+00:00
[2022-11-27T16:01:43.970+0000] {logging_mixin.py:137} INFO - {'code': 400, 'message': 'The <code>regionCode</code> parameter specifies an invalid region code.', 'errors': [{'message': 'The <code>regionCode</code> parameter specifies an invalid region code.', 'domain': 'youtube.parameter', 'reason': 'invalidRegionCode', 'location': 'regionCode', 'locationType': 'parameter'}]}
[2022-11-27T16:01:43.971+0000] {logging_mixin.py:137} INFO - Writing I data to file...
[2022-11-27T16:01:44.156+0000] {logging_mixin.py:137} INFO - {'code': 400, 'message': 'The <code>regionCode</code> parameter specifies an invalid region code.', 'errors': [{'message': 'The <code>regionCode</code> parameter specifies an invalid region code.', 'domain': 'youtube.parameter', 'reason': 'invalidRegionCode', 'location': 'regionCode', 'locationType': 'parameter'}]}
[2022-11-27T16:01:44.156+0000] {logging_mixin.py:137} INFO - Writing D data to file...
[2022-11-27T16:01:44.157+0000] {python.py:177} INFO - Done. Returned value was: None
[2022-11-27T16:01:44.173+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=dag_youtube_api, task_id=getDataToLocal, execution_date=20221127T160141, start_date=20221127T160143, end_date=20221127T160144
[2022-11-27T16:01:44.247+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2022-11-27T16:01:44.280+0000] {taskinstance.py:2623} INFO - 0 downstream tasks scheduled from follow-on schedule check
